schemas/procedures.json

```diff
@@ -43,6 +43,12 @@
             "uniqueItems": true,
             "title": "Injections"
         },
+        "fiber_implants": {
+            "type": "array",
+            "items": {"$ref": "#/$defs/impant"},
```

Comment:
I think this is a typo (should be implant)

Reply:
thanks - and sorry!